benchmarks/bench_20newsgroups.py

```diff
@@ -46,18 +46,18 @@
 
     print("20 newsgroups")
     print("=============")
-    print("X_train.shape = {0}".format(X_train.shape))
-    print("X_train.format = {0}".format(X_train.format))
-    print("X_train.dtype = {0}".format(X_train.dtype))
-    print("X_train density = {0}"
-          "".format(X_train.nnz / np.product(X_train.shape)))
-    print("y_train {0}".format(y_train.shape))
-    print("X_test {0}".format(X_test.shape))
-    print("X_test.format = {0}".format(X_test.format))
-    print("X_test.dtype = {0}".format(X_test.dtype))
-    print("y_test {0}".format(y_test.shape))
+    print(f"X_train.shape = {X_train.shape}")
+    print(f"X_train.format = {X_train.format}")
+    print(f"X_train.dtype = {X_train.dtype}")
```

Comment:
I don't remember which Python version this was introduced (so IDK if we actually support it), but these are equivalent to `print(f"{X_train.shape = }")`. This will print e.g. `X_train.shape = 123`.

Reply:
I think it's Python 3.8 so we cannot use that yet in scikit-learn.